src/pages/LoginPage.js

```diff
@@ -1,9 +1,44 @@
 import React from 'react'
+import {ThemeButton} from "../components/ThemeButton";
+import { useForm } from "react-hook-form";
+import {TextField, Container, Grid, Box, Link} from "@material-ui/core";
+import { makeStyles } from '@material-ui/core/styles';
+import { Link as RouterLink } from 'react-router-dom';
+import {login} from "../mock/auth";
 
 export const LoginPage = () => {
+    const { register, handleSubmit, errors } = useForm();
+    const onSubmit = data => {
+        console.log(data);
+        login();
+    }
+    const classes = useStyles();
+
   return (
-    <div>
-      LoginPage
-    </div>
+      <Container className={classes.container} maxWidth="xs">
+          <form onSubmit={handleSubmit(onSubmit)}>
+              <Grid container
+                    direction="column"
+                    justify="space-between"
+                    alignItems="center"
+              >
+                  <Box><TextField label="Username" name="usernameLoginField" inputRef={register( {required: true})} /></Box>
+                  <Box marginTop={2}><TextField label="Password" name="passwordLoginField" inputRef={register({ required: true })} type="password" /></Box>
```

Comment:
I stedet for å sette `required` til å være `true` kan du bare skrive inn feilmeldingen "Du må gi et brukernavn!". Deretter kan du legge til følgende props til `<TextField ... >`:
```
error={!!errors.usernameLoginField }
helperText={errors.usernameLoginField ? errors.usernameLoginField.message : ""}
```

Da vil feilmeldingene komme rett under feltet og det vil bli merket rødt. Tror det kan bli litt ryddigere. 

I tillegg blir det lettere å legge til flere feilmeldinger uten å ha de hardkodet et annet sted. For eksempel går det an å legge til minLength slik: `{ required: "Du må gi et brukernavn!", minLength: { value: 8, message: "Minimum passord lengde er 8" }}`

Reply:
Tror ikke heelt jeg skjønner hva du mener her. Har byttet ut boolean med teksten men skjønte ikke helt det siste. Kan du lage et forslag? 😬

Reply:
Eller jo nå ser jeg det. Jeg skal legge til de propsene til selve tekstfeltet. 🙄 Men hva er greien med `{!!errors.username}` ? 🙈

Reply:
Ja, stemmer at propsene skal i Textfield, litt dårlig forklart av meg😋. De doble ! gjør et objekt eller en tekst om til en ekte boolsk verdi. Hvis det er noen errors assosiert med username blir den true og aktiverer error-utseende 😊